cli/man/transact-playlist-submit.1.md

```diff
@@ -51,7 +51,7 @@ OPTIONS
   sign the batches as well as generate a JWT for authentication.
 
 `--rate RATE`
-: How many batches to submit per second, ex `5`. (default: `1`)
+: How long to wait between batch submits, ex `5s`. (default: `1s`)
```

Comment:
This should also have the explanation of the time interval format

Reply:
fixed/expanded upon